pages/_app.js

```diff
@@ -31,9 +31,9 @@ function PlugRoot({ Component, pageProps }) {
         <meta property="twitter:title" content="Plug for Mac" />
         <meta
           property="twitter:description"
-          content="OS X player for The Hype Machine ♥"
+          content="macOS player for the Hype Machine :heart:"
```

Comment:
I don't think these will render correctly, you should use the emoji directly instead :v: